mpf/platforms/interfaces/light_platform_interface.py

```diff
@@ -95,6 +95,8 @@ def set_fade(self, start_brightness, start_time, target_brightness, target_time)
             self.task = self.loop.create_task(self._fade(start_brightness, start_time, target_brightness, target_time))
             self.task.add_done_callback(Util.raise_exceptions)
         else:
+            if self.task:
+                self.task.cancel()
```

Comment:
We should set task to None here or we might double cancel it. Otherwise this might raise an exception. Not sure if it can already be done which might cause an exception as well. For that we would have to check task.done.